content/w/andrzej-supron.md

```diff
@@ -1,8 +1,70 @@
 +++
 title = "Andrzej Supron"
 template = "talent_page.html"
+authors = ["Sewi The Referee"]
 [extra]
 toclevel = 2
 [taxonomies]
 country = ["PL"]
 +++
+
+Andrzej Supron is a Polish Greco-Roman wrestler, Olympian and wrestling commentator best known from Extreme Sports Channel. He is also a former wrestling promoter, and one of the pioneers of bringing pro wrestling to Poland.
+
+## Basic info
+
+* Years active: 1990s-present
+* Gimmicks/nicknames:
+  - Andrzej Supron {{ org_badge(orgs=['wws', 'tbw', 'ptw']) }}
+
+## Professional wrestling career
+
+### Supron Stars (1990-1992)
+
+In early 1990s Andrzej Supron organised a group of bodybuilders, sportsmen and actors trained loosely by Supron to perform in Poland and ex-Soviet republics such as Georgia, Armenia, Lithuania etc. However, not much information or media is avaliable from that time. 
+
+### World Wrestling Superstars (1993)
+
+In 1993 Andrzej Supron contacted a group of American wrestlers touring Europe, called [World Wrestling Superstars](@/o/wws.md), and convinced them to perform a single [show](@/e/wws/1993-12-14-wws-world-wrestling-superstars.md) in Poland. It took place in December 1993 at Torwar arena in Warsaw. This was the only international wrestling event held in Poland before [WWE](@/o/wwe.md) arrived in the mid-2010.
+
+### Polish Pro-Wrestling Federation (1999-2003)
+
+Around 1999 [Paweł Borkowski](@/w/pawel-borkowski.md) came to Andrzej Supron with a plan to create the first pro wrestling federation in the country, to be named simply "Polish Pro-Wrestling Federation", or PPWF. Unable to secure international names, the organization had to rely only on home-grown talent. While they did manage to secure sponsorship and a deal to buy a professional wrestling ring, no events were actually held due to low interest and poor ticket sales. A second attempt in 2003 secured support from Belgian wrestler and promoter Bernard Vandamme, but once again the event was cancelled, just three weeks before the scheduled date.
+
+### Total Blast Wrestling (2009-2011)
+
+#### Split from Do Or Die Wrestling
```

Comment:
Instead of copying the text, I prefer condensing it and leaving a link to the main article. So for this section, trim it down to only mention the cooperation and breakup with Roid, then successfully holding two shows. After the header, add a link to TBW with text like "Main article: Total Blast Wrestling".